packages/devtools_app/lib/src/shared/preferences/preferences.dart

```diff
@@ -149,9 +151,11 @@ class PreferencesController extends DisposableController
     );
     toggleVmDeveloperMode(vmDeveloperModeValue);
     addAutoDisposeListener(vmDeveloperModeEnabled, () {
-      storage.setValue(
-        _UiPreferences.vmDeveloperMode.storageKey,
-        '${vmDeveloperModeEnabled.value}',
+      safeUnawaited(
+        storage.setValue(
```

Comment:
do we ever await storage.setValue? If not, we could change the return type of setValue to void and then add the safeUnawatied wrapper inside that method.

Reply:
It looks like we were awaiting `storage.setValue` in some places though. My comment was only if we weren't  awaiting this method call anywhere but it looks like we were. I don't know if we want to change the behavior of places where we were awaiting.